crowdsourcing/viewsets/google_drive.py

```diff
@@ -76,4 +78,31 @@ def auth_end(self, request):
         except Exception as e:
             message = 'Something went wrong.'
 
-        return HttpResponseRedirect('/')
\ No newline at end of file
+        return HttpResponseRedirect('/')
+
+class GoogleDriveUtil(APIView):
+    '''
+    Reference: https://developers.google.com/drive/v2/reference/children/list
+    Creates a Google Drive API service object and outputs the names and IDs
+    for files in google drive.
+    '''
+    def list_DriveContents(credentials):
+        credentials = Credentials.new_from_json(credentials)
+        http = credentials.authorize(httplib2.Http())
```

Comment:
build moves to Class init so that we can use it in other methods, we always need it